client/rpmtrans.c

```diff
@@ -576,8 +576,7 @@ TDNFTransAddInstallPkg(
     {
         BAIL_ON_TDNF_RPM_ERROR(dwError);
```

Comment:
The point here is this BAIL_ON_TDNF_RPM_ERROR(dwError); can return back if dwError is 0. Then code need not need go in else path.

Reply:
Ahh, I see, thanks. But this else was an odd way of catching that cass. Would it have been cleaner to say,
```c
if (dwError) {
  if (dwError != RPMRC_NOTTRUSTED && dwError != RPMRC_NOKEY) {
    goto error;
  }
  if (nGPGSigCheck) {
     ...
  }
}
```
This construct distinguishes errors path from main code flow. But that's behind us now :-)